adapters/criteo/criteo_slot_id_generator.go

```diff
@@ -0,0 +1,22 @@
+package criteo
```

Comment:
Looking good. My only comment is to consider a simpler file name. You may want to remove the 'criteo` prefix for simplicity and to match with the `models.go` naming.

Reply:
I went for `generators.go` as filename